lib/vagrant-service-manager/services/open_shift.rb

```diff
@@ -33,8 +33,16 @@ def self.info(machine, ui, options = {})
       end
 
       def self.print_info(ui, options)
-        label = 'default'
-        label = 'script_readable' if options[:script_readable]
+        label = if options[:script_readable]
```

Comment:
Could you extract this part into `PluginUtil` module and so that it become scalable and could be used in future providers or services.

```
def self.env_label(script_readable)
  if script_readable
    'script_readable'
  elsif OS.unix?
    'non_windows'
  elsif OS.windows_cygwin?
   'windows_cygwin'
  else
   'windows'
  end
end
```